cmd/oteltestbedcol/builder-config.yaml

```diff
@@ -17,7 +17,7 @@ extensions:
 
 exporters:
   - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.118.1-0.20250123125445-24f88da7b583
-  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.118.1-0.20250123125445-24f88da7b583
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.119.0
   - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.118.1-0.20250123125445-24f88da7b583
```

Comment:
(question but not blocking the PR) why are these not updated in the release process